### PR TITLE
COMMERCE-4186

### PR DESCRIPTION
**Pull request by dsitu**

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dannysitu** [comment]
ci:test:sf

**Comment by dantewang** [comment]
@dsitu It's OK. SF fails because a typo in one of Lily's commit messages.

File: modules/apps/commerce/commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/service/impl/CommercePriceListLocalServiceImpl.java [hunk 1]
```diff
@@ -570,7 +570,11 @@ public void checkCommercePriceLists() throws PortalException {
 
 	@Override
 	public void cleanPriceListCache(long companyId) {
-		_multiVMPool.removePortalCache("PRICE_LISTS_" + companyId);
+		PortalCache<String, Serializable> portalCache =
```
Comment (dantewang) [inline modules/apps/commerce/commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/service/impl/CommercePriceListLocalServiceImpl.java:573]: @dsitu Should this change be included in the SampleSQLBuilder pull?

File: benchmarks/benchmarks.properties [hunk 2]
```diff
@@ -79,7 +79,7 @@
     #
     # Specify the number of commerce sites to generate.
     #
-    sample.sql.max.commerce.group.count=1
+    sample.sql.max.commerce.group.count=0
```
Comment (dantewang) [inline benchmarks/benchmarks.properties:82]: @dsitu The commit message is to "disable commerce product attachment" but it seems the whole commerce generation is disabled. Maybe it should be splitted into different commits?

File: benchmarks/benchmarks.properties [hunk 3]
```diff
@@ -89,7 +89,7 @@
     #
     # Specify the number to set commerce warehouse item quantities.
     #
-    sample.sql.max.commerce.inventory.warehouse.item.qunantity=10
+    sample.sql.max.commerce.inventory.warehouse.item.quantity=10
```
Comment (dantewang) [inline benchmarks/benchmarks.properties:92]: @dsitu This should be merged into https://github.com/dantewang/liferay-portal/pull/2002/commits/9ca59a837e55705ffe46b77ce0659412fbbfd056

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by dantewang** [comment]
@dsitu After making the above changes, please send to @slnn and Lily will double check the patterns, then we can forward this up. Thanks!

**Pull request closed without merging**